src/main/java/com/epam/prd/jjop/helpers/arrays/Sort.java

```diff
@@ -0,0 +1,103 @@
+package com.epam.prd.jjop.helpers.arrays;
+
+import java.util.Arrays;
+import java.util.Comparator;
+
+/**
+ * Examples of sorting an array using methods from Arrays class.
+ *
+ * @author Anna Silenko
+ * @see java.util.Arrays
+ */
+class Sort {
+    public static void main(String[] args) {
+        /**
+         * Let's prepare some arrays to sort:
+         */
+        Cat noisyCat = new Cat("Noisy cat", 1, 16);
+        Cat sleepingCat = new Cat("Sleeping cat", 2, 7);
+        Cat purringCat = new Cat("Purring cat", 6, 2);
+        Cat[] cats = {sleepingCat, purringCat, noisyCat};
+
+        Dog goodBoy = new Dog("Good boy", 1);
+        Dog badBoy = new Dog("Bad boy", 10);
+        Dog fafik = new Dog("Fafik", 3);
+        Dog[] dogs = {fafik, badBoy, goodBoy};
+
+        int[] simpleArrayToSort = {7, 6, 9, 3, 5, 2, 1, 10, 8, 4};
+
+        /**
+         * <p>Arrays class provides several overloaded methods for sorting an Array.</p>
+         * <p><b>Default sort order is ascending.</p></b>
+         * <p>If we want, we can only sort a specific range of the array with {@link java.util.Arrays#sort(int[], int, int)}
+         * As usual the first index is inclusive and the second one is exclusive.</p>
+         */
+        System.out.println("simpleArrayToSort before sorting: " + Arrays.toString(simpleArrayToSort));
+        Arrays.sort(simpleArrayToSort, 2, 6);
+        System.out.println("simpleArrayToSort after sorting in range [2, 6): " + Arrays.toString(simpleArrayToSort));
+
+        /**
+         * We can simply skip the range parameters and sort the entire array with {@link Arrays#sort(int[])}
+         */
+        Arrays.sort(simpleArrayToSort);
+        System.out.println("simpleArrayToSort after sorting: " + Arrays.toString(simpleArrayToSort));
+
+        /**
+         * Arrays of Object are sorted according to the natural ordering of its elements. Natural ordering use
+         * {@link Comparable#compareTo(Object)}
+         * So to be able to sort an array of objects using this method, a class that wraps these objects must implement
+         * Comparable.
+         */
+        //Arrays.sort(dogs); //this will throw an Exception, because Dog do not implement Comparable.
+
+        /**
+         * According to Cat implementation, natural ordering compare Cat's age. After sorting Cats should be ordered
+         * from youngest to oldest.
+         */
+        System.out.println("cats before sorting: " + Arrays.toString(cats));
+        Arrays.sort(cats);
+        System.out.println("cats after sorting using natural order: " + Arrays.toString(cats));
```

Comment:
If the sorting mechanism uses cat's age field, then wouldn't it be more clear if the toString method showed not only cat's name, but also the age? Something like `return "" + name + " (" + age + ")";`